### ISSUE TITLE
Steal Tools warning when using Node 5

### ISSUE THREAD

**Issue by Macrofig** [stealjs/steal-tools#369]
I'm not sure if we just didn't update the `engineDependencies` or if this is a specific limitation. Either way, I think it needs addressing.

```
npm WARN engine steal-tools@0.13.3: wanted: {"node":"0.10.x - 0.12.x || ^3.0.0"} (current: {"node":"5.2.0","npm":"3.3.12"})
npm WARN engine steal-tools@0.13.3: wanted: {"node":"0.10.x - 0.12.x || ^3.0.0"} (current: {"node":"5.2.0","npm":"3.3.12"})
```

**Comment by Macrofig** [comment]
Just realized I duplicated this issue #361. Closing in favor of that issue.

**Comment by matthewp** [comment]
No, that's something different, this is the engines field: https://github.com/stealjs/steal-tools/blob/master/package.json#L102  It should be updated

**Comment by pYr0x** [comment]
fixed in minor branch and pre-release 0.16
